test/functional/driver/driver-e2e-specs.js

```diff
@@ -293,6 +294,47 @@ describe('XCUITestDriver', function () {
         res.events.newSessionStarted[0].should.be.above(res.events.newSessionRequested[0]);
       });
     });
+    describe('w3c compliance', () => {
+      const sessionUrl = `http://${HOST}:${PORT}/wd/hub/session`;
+      it('should accept w3c formatted caps', async function () {
+        const { status, value, sessionId } = await request.post({url: sessionUrl, json: W3C_CAPS});
+        status.should.equal(0);
```

Comment:
java client has the following verification to confirm the response is W3C-compliant:

```java
  if (tuple.getData().containsKey("status")) {
      return Optional.empty();
    }
```

how we are going to handle this?

Reply:
@mykola-mokhnach I'll update `appium-base-driver` so that it doesn't include the `status` key.

I hadn't bothered removing it because I didn't think that it would cause any issues. But taking it out is fine, just one conditional.